src/lib/utils/string_functions.cpp

```diff
@@ -0,0 +1,18 @@
+#include <boost/algorithm/string/split.hpp>
```

Comment:
Maybe call this file "string_utils.hpp" since we already have a few files with that suffix?
(Most translation units contain functions, so that name isn't really speaking)